cli/Main.ml

```diff
@@ -15,7 +15,7 @@ module J = JSON
  *
  * related:
  *  - https://astexplorer.net/, supports many languages, many parsers
- *)
+*)
```

Comment:
I know this is one is weird. I filed an issue. I might fix it myself. See https://github.com/OCamlPro/ocp-indent/issues/312